src/components/CriteriaList/CriteriaTable.tsx

```diff
@@ -53,15 +54,11 @@ const CriteriaTable: FC = () => {
                 >
                   Edit
                 </Button>
-
-                <Button
-                  color="secondary"
-                  size="small"
-                  startIcon={<FontAwesomeIcon icon={faTrashAlt} />}
-                  onClick={(): void => deleteCriteria(c.id)}
-                >
-                  Delete
-                </Button>
+                <ConfirmedDeletionButton
+                  deleteType="criterion"
+                  deleteName={c.label}
+                  deleteMethod={(): void => deleteCriteria(c.id)}
```

Comment:
This causes an extra re-render since the function is always the same but every time the CriteriaTable re-renders a new anonymous function is created. Make sure the function passed in a callback.
![Screen Shot 2020-07-17 at 2 43 22 PM](https://user-images.githubusercontent.com/53485517/87820317-df1bfb00-c83b-11ea-88c0-90b37da257bc.png)